Source/Csla.AspNetCore/Blazor/ApplicationContextManagerInMemory.cs

```diff
@@ -168,18 +170,16 @@ private void SetHostPrincipal(Task<AuthenticationState> task)
     /// <summary>
     /// Gets the local context.
     /// </summary>
-    public IContextDictionary GetLocalContext()
+    public IContextDictionary? GetLocalContext()
```

Comment:
The getter should create an instance on demand so the end developer never gets a null, even if the private value was null behind the scenes.

Reply:
Good catch. I first tried to built it without `null` but reverted that. Seems like not fully.